### DISCUSSION TITLE
Put a window as a child of another window

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1021] [Q&A] [answered]
I have the following starting layout:

![mpv-shot0001](https://user-images.githubusercontent.com/54762742/201759150-4a298f8c-a178-4c3c-a169-911da291eaa0.jpg)

I have to end up with the following layout:

![mpv-shot0002](https://user-images.githubusercontent.com/54762742/201759192-3c37bd52-2bd3-406f-9d5e-72953e0555f3.jpg)

So "Window 3" must switch from being a child of Window 1 to being a child of Window 4.

I'm using the dwindle layout.

What is the shortest way to do that?

I ended up focusing on Windows 4, creating a new child, swapping this child with "Window 3", and then deleting it, as the following video shows:

![output](https://user-images.githubusercontent.com/54762742/201760451-86e667a8-a72d-4ae2-ba8e-8d3c684bb847.gif)

**Comment by vaxerski** [comment, marked as answer]
mouse binds (dragging the window)

**Reply by raffaelemancuso** [reply]
that would defeat my evil purpose of using only the keyboard 😛

**Reply by vaxerski** [reply]
unlucky. It's planned.

**Reply by raffaelemancuso** [reply]
but there must be a command line with `hyprctl` that I can bind, right?

**Reply by vaxerski** [reply]
nope. The dispatcher atm only swaps windows
